### ISSUE TITLE
Incorrect Capitalization on the 18F Accessibility Guide Writing Guide Page

### ISSUE THREAD

**Issue by jfc3-dol** [18F/accessibility#300]
On the 18F Accessibility Guide Writing Guide page ( https://accessibility.18f.gov/writing-guide/ ), the second occurrence of the word "section" in the sentence below should be capitalized.

We conform with the standards of **Section** 508, not the law of **section** 508.
